common.pri

```diff
@@ -2,6 +2,10 @@
 #
 # To use it, define ROOT_DIR variable and include($$ROOT_DIR/common.pri)
 
+# We use some library features that were introduced in Mac OS X 10.8.
+# Qt5.6.x sets target OS X version to 10.7 which leads to compile errors.
+# TODO: Remove when qt5.7 is available in brew.
```

Comment:
This one is even closer: http://dictionary.cambridge.org/ru/%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0/%D0%B1%D1%80%D0%B8%D1%82%D0%B0%D0%BD%D1%81%D0%BA%D0%B0%D1%8F-%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0/when